### ISSUE TITLE
Call to private method RainLoop\Actions::getAccountFromToken() error

### ISSUE THREAD

**Issue by alexzeitgeist** [RainLoop/rainloop-webmail#638]
Just upgraded from 1.8.3.304 to 1.9.0.322. After the upgrade, I cannot login anymore. Instead, when I enter the correct login credentials, the screen would refresh and the login prompt would show again. This is part of the error message from the webserver:

[error] 17528#0: *1 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Call to private method RainLoop\Actions::getAccountFromToken() from context '' in /var/www/mail/rain
/rainloop/v/1.9.0.322/app/libraries/RainLoop/Actions.php on line 9140" while reading response header from upstream...

**Comment by RainLoop** [comment]
Thanks! Sorry for that.

Fast fix:

`/var/www/mail/rain/rainloop/v/1.9.0.322/app/libraries/RainLoop/Actions.php` ~1274

```php
private function getAccountFromToken($bThrowExceptionOnFalse = true)
{
```

```php
public function getAccountFromToken($bThrowExceptionOnFalse = true)
{
```

**Comment by alexzeitgeist** [comment]
Fastest fix ever. Thanks.  :+1:

**Comment by RainLoop** [comment]
Fixed in 1.9.0.323 version.
